packages/use-dataloader/src/__tests__/useDataLoader.tsx

```diff
@@ -582,4 +608,73 @@ describe('useDataLoader', () => {
     expect(result.current[0].isSuccess).toBe(true)
     expect(mockedFn).toBeCalledTimes(2)
   })
+
+  test('should cancel request', async () => {
+    const onCancel = jest.fn()
+    const { result, unmount } = renderHook<
+      UseDataLoaderHookProps,
+      UseDataLoaderResult
+    >(
+      props =>
+        useDataLoader(
+          props.key,
+          () => {
+            const promise = new Promise(resolve => {
+              setTimeout(() => resolve(true), 500)
```

Comment:
I know it already was the value used for other tests, but 500ms seems too long IMHO.

Reply:
Changed by 100ms that's better yes